app/models/crime_application.rb

```diff
@@ -97,33 +97,4 @@ def client_details_complete?
   def passporting_benefit_complete?
     valid?(:passporting_benefit)
   end
-
-  def applicant_requires_nino_evidence?
-    # the applicant is over 18 and no NINO has been entered
-    # plus any one or more of the following apply:
-    # - they are passported on means
-    # - they receive any non-passporting benefit
-    # - the case type is `indictable` or `already_in_crown_court`
-    return false unless applicant_18_or_over_at_date_stamp? && applicant&.nino.blank?
-
-    case_types = [
-      CaseType::INDICTABLE.to_s,
-      CaseType::ALREADY_IN_CROWN_COURT.to_s
-    ]
-
-    applicant.has_passporting_benefit? || case_types.include?(self.case&.case_type) || income_benefits.any?
-  end
-
-  # rubocop:disable Metrics/AbcSize
```

Comment:
#applicant_18_or_over_at_date_stamp? has been moved to Person, simplified and time zone bug fixed.